src/libpgagroal/configuration.c

```diff
@@ -178,6 +178,46 @@ pgagroal_init_configuration(void* shm)
    return 0;
 }
 
+/**
+ *
```

Comment:
Remove comments, leave only in the header file or when a static function is defined.